src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const renderAge = () => {
+    if (person.age) {
+      return <p className="Person__age">I am {person.age}</p>;
+    }
+
+    return null;
+  };
```

Comment:
Here you can use the ternary operator.
```suggestion
  const renderAge = () => {
    return age ? <p className="Person__age">I am {age}</p> : null;
  };
```